mergify_engine/actions/assign.py

```diff
@@ -26,28 +28,41 @@
 
 class AssignAction(actions.Action):
     validator = {
+        # NOTE: "users" is deprecated, but kept as legacy code for old config
         voluptuous.Required("users", default=[]): [
             voluptuous.Coerce(types.Jinja2, types.GitHubLogin)
-        ]
+        ],
+        voluptuous.Required("add_users", default=[]): [
+            voluptuous.Coerce(types.Jinja2, types.GitHubLogin)
+        ],
     }
 
     silent_report = True
 
     def run(self, ctxt: context.Context, rule: rules.EvaluatedRule) -> check_api.Result:
-        wanted = set()
-        for user in set(self.config["users"]):
-            try:
-                user = ctxt.pull_request.render_template(user)
-            except context.RenderTemplateFailure as rmf:
-                # NOTE: this should never happen since the template is validated when parsing the config 🤷
-                return check_api.Result(
-                    check_api.Conclusion.FAILURE, "Invalid assignee", str(rmf)
-                )
-            else:
-                wanted.add(user)
+        # NOTE: "users" is deprecated, but kept as legacy code for old config
+        if self.config["users"]:
+            return self._add_assignees(ctxt, self.config["users"])
+
+        if self.config["add_users"]:
+            return self._add_assignees(ctxt, self.config["add_users"])
+
+        return check_api.Result(
+            check_api.Conclusion.SUCCESS,
+            "",
+            "",
```

Comment:
You can still add some text here. It's not going to be sent to the users, because the action has `silent_report = True`, but it's still good to see it in the logs.